ledger/ledger-common/src/main/kotlin/net/corda/v5/ledger/common/transaction/PrivacySalt.kt

```diff
@@ -0,0 +1,12 @@
+package net.corda.v5.ledger.common.transaction
+
+import net.corda.v5.base.annotations.CordaSerializable
+
+/**
+ * A privacy salt is required to compute nonces per transaction component in order to ensure that an adversary cannot
+ * use brute force techniques and reveal the content of a Merkle-leaf hashed value.
+ */
+@CordaSerializable
```

Comment:
Hmm. 
If I remove `@CordaSerializable` from `Party`. https://github.com/corda/corda-api/pull/487/files#diff-220f68872e64df2e2d86100c1110074e2e35665a0d012096a095a16e87cf675aR13
Whose implementation does not have it either. https://github.com/corda/corda-runtime-os/pull/1715/files#diff-5952c829255e7e76a07adc3fe7ae1a9c4a736fbfa54c4cd151cee4167aa05661R7
This test: https://github.com/corda/corda-runtime-os/pull/1715/files#diff-f4502fe9888db2f9bd6bfec08904c1ec06978d53e18ab85845626fcbf5ac2363R71
fails with this:
```
Class "interface net.corda.v5.ledger.consensual.Party" is not annotated with @CordaSerializable.
java.io.NotSerializableException: Class "interface net.corda.v5.ledger.consensual.Party" is not annotated with @CordaSerializable.
...
```
Otherwise it works.
So it looks like doing something in a similar case.

Reply:
(Since that `Party` got a custom serialiser. Hopefully the links survive the related pushes...)